R/rcpp.R

```diff
@@ -6,28 +6,20 @@
 #'   * May create an initial placeholder `.c` or `.cpp` file
 #'   * Creates `Makevars` and `Makevars.win` files (`use_rcpp_armadillo()` only)
 #'
-#' @param name If supplied, creates and opens `src/name.{c,cpp}`.
-#'
-#' @details
-#'
-#' When using compiled code, please note that there must be at least one file
```

Comment:
This weird wrinkle (implemented in `use_src_example_script())` is no longer needed because we always create a file.